UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/Helper/ExchangeSetApiClient.cs

```diff
@@ -57,70 +58,157 @@ public async Task<HttpResponseMessage> GetExchangeSetBasedOnDateTimeAsync(string
         /// <param name="productVersionModel"></param>
         /// <param name="callbackUri">callbackUri, pass NULL to skip call back notification</param>
         /// <param name="accessToken">Access Token, pass NULL to skip auth header</param>
+        /// <param name="isUnencrypted"></param>
```

Comment:
Please change this as below in each places in this class.
```suggestion
        /// <param name="isUnencrypted">isUnencrypted, pass true for unencrypted and false for encrypted exchange set</param>
```

Reply:
Resolved. Thanks